### PR TITLE
[#259] Add a question to determine if the beta user is a legal professional

### PR DESCRIPTION
**Pull request by lancelafontaine**
[WORK-IN-PROGRESS]

Client needs to be updated to support asking an additional question. Exact text TBD.

 [#259] 
- [x] Updates Beta backend server to support storing whether a user is a legal professional
- [x] Updates Beta backend server API documentation
- [x] Updates Beta client

### CODE REVIEW

File: src/beta_server/README.md [hunk 1]
```diff
@@ -82,5 +86,31 @@ Updates a user's subscription status based on their ID. `1` is subscribed, `0` i
 - `415` if request does not contain valid JSON
 - `422` if the `is_subscribed` key is not present
 - `422` if the `is_subscribed` key is not an integer
-- `422` if the `id` key is not present
+
+### `PUT /legal`
```
Comment (dodoels) [inline src/beta_server/README.md:90]: i found `legal` is a bit confused. Not a critical issue.

File: src/beta_server/README.md [hunk 2]
```diff
@@ -57,12 +60,13 @@ Updates a user's email address based on their ID.
 - `415` if request does not contain valid JSON
 - `422` if the `email` key is not present
 - `422` if the `email` value is too long
-- `422` if the `id` key is not present
 
 ### `PUT /subscription`
```
Comment (dodoels) [inline src/beta_server/README.md:64]: Do we still need this endpoint?

**Review by dodoels — APPROVED** [review]

**Comment by dodoels** [comment]
Is this PR blocked?

**Review by Vynny — APPROVED** [review]
LGTM

**Comment by lancelafontaine** [comment]
Blocked by client-side. Need to change text and order of questions in beta.

**Review by arekmano — COMMENTED** [review]
LGTM, but why is it an integer instead of a boolean?

**Comment by dodoels** [comment]
image
image

**Comment by dodoels** [comment]
Integration is done, ready for merge

**Pull request merged**